### ISSUE TITLE
Mame2003 - Mouse Window Auto-Focus on lightgun games

### ISSUE THREAD

**Issue by shaylevary** [emulatorjs/emulatorjs#664]
Hi,

The new Mame2003 core finally support light gun games (Operation Wolf, Lethal Enforcer, ect...).

I've encountered  2 issues while playing some of the LightGun games:

1. It is necessary to disable the mouse cursor and activate Mouse Auto-Focus mode inside the game window in order to support LightGun games (on mobile, on the other hand, it works fine).

2. Another less critical issue, in some games it is necessary to use the right mouse button (for example in Operation Wolf it allows you firing missiles).
Currently the right mouse button is used to display the emulatorjs menu, so i have to use the mouse left button for shooting and a keyboard button for firing missles.

3. Is it possible to use Mame's cheats menu to activate in-game cheats?
Because for example in the game "Lethal Enforcer" I am required to move the cursor to the bottom of the screen every time i would like to reload my ammunition.
If i will be able to use cheats, i can set it to Auto-Reload.
Another option is to set a keyboard key or right mouse button to perform the reload function - instead of having to scroll-down to the bottom of the screen every time i would like to reload.
![lethalenforcer2](https://github.com/EmulatorJS/EmulatorJS/assets/10597517/207c91b4-ecfb-44ba-8919-bfd519c73e89)

**Comment by ethanaobrien** [comment]
Is this still an issue

**Comment by shaylevary** [comment]
Yes, any mame2003 game that uses a lightgun with the mouse should have a mouse focus option, so that the mouse arrow disappears when clicking on the game window to focus on it.
Maybe you should add an option similar to "ejs_mousefocus = true" to set the mouse focus option for any specific game.

![20250112_232641](https://github.com/user-attachments/assets/d7eeef9e-1ea2-4021-9ce1-d4a36bdab14a)
